.github/workflows/ci.yml

```
@@ -0,0 +1,133 @@
+      - "fix-*"
+  pull_request:
+    branches:
+      - "main"
+      - "phase-*"
+
+jobs:
+  # Job 1: Check Quality (Python & Dockerfile)
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.10)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)